packages/amplify-provider-awscloudformation/src/utils/upload-auth-trigger-template.ts

```diff
@@ -23,17 +23,6 @@ export async function uploadAuthTriggerTemplate(context: $TSContext) {
     };
   }
   return {
-    AuthTriggerTemplateURL: await uploadCfnToS3(context, S3_UPLOAD_PATH, cfnObject),
+    AuthTriggerTemplateURL: await uploadTemplateToS3(context, resourceDir, AUTH_TRIGGER_TEMPLATE, category),
```

Comment:
won't the trigger template already be uploaded when `updateS3Templates` in `push-resources.ts` runs? If so, can we get the url from the providerMetadata that is stored rather than re-uploading?